internal/backup_list_test.go

```diff
@@ -23,22 +23,22 @@ func TestBackupListFlagsFindsBackups(t *testing.T) {
 
 var backups = []internal.BackupTime{
 	{
-		BackupName:  "base_123",
-		Time:        time.Date(2019, 4, 25, 14, 48, 0, 0, time.UTC),
-		WalFileName: "ZZZZZZZZZZZZZZZZZZZZZZZZ",
+		BackupName:       "base_123",
+		ModificationTime: time.Date(2019, 4, 25, 14, 48, 0, 0, time.UTC),
+		WalFileName:      "ZZZZZZZZZZZZZZZZZZZZZZZZ",
 	},
 	{
-		BackupName:  "base_456",
-		Time:        time.Date(2018, 7, 5, 1, 1, 50, 0, time.UTC),
-		WalFileName: "ZZZZZZZZZZZZZZZZZZZZZZZZ",
+		BackupName:       "base_456",
+		ModificationTime: time.Date(2018, 7, 5, 1, 1, 50, 0, time.UTC),
+		WalFileName:      "ZZZZZZZZZZZZZZZZZZZZZZZZ",
 	},
 }
 
 func TestBackupListCorrectOutput(t *testing.T) {
 	const expected = "" +
-		"name     last_modified        wal_segment_backup_start\n" +
-		"base_456 2018-07-05T01:01:50Z ZZZZZZZZZZZZZZZZZZZZZZZZ\n" +
-		"base_123 2019-04-25T14:48:00Z ZZZZZZZZZZZZZZZZZZZZZZZZ\n"
```

Comment:
Changing the time format is undesirable unless there is some strong reason to do it.

Reply:
Yes, I didn't notice it.